### ISSUE TITLE
yolox_head_fast.py中当G == 0的返回loss有问题

### ISSUE THREAD

**Issue by xiaoxuebajie** [miemie2013/miemiedetection#22]
`
if G == 0:  # 所有图片都没有gt时
            obj_targets = torch.zeros((N, A, 1), dtype=torch.float32, device=obj_preds.device)
            num_fg = 1  # 所有图片都没有gt时，设为1
            loss_obj = self.bcewithlog_loss(obj_preds, obj_targets)
            loss_obj = loss_obj.sum() / num_fg
            losses = {
                "loss_obj": loss_obj,
            }
            return losses
` 
yolox_head_fast.py中的这一段是不是错了
这个losses和下面的对不上
`            return (
                loss,
                reg_weight * loss_iou,
                loss_obj,
                loss_cls,
                loss_l1,
                num_fg / max(num_gts, 1),
            )`
